lib/gitlib.sh

```diff
@@ -258,7 +322,108 @@ gitlib::repo_state () {
     logecho "$FAILED"
     logecho
     logecho "$TOOL_ROOT is not up to date."
-    logecho "$ git pull # to try again"
+    logecho "$ git pull"
     return 1
   fi
 }
+
+# Set up git config for GCB
+if ((FLAGS_gcb)); then
+  gitlib::git_config_for_gcb || common::exit "Exiting..."
+fi
+
+###############################################################################
+# Search for a matching release tracking issue
+# @param version RELEASE_VERSION_PRIME
+# returns 1 if none found
+# prints most recent issue maching
+gitlib::search_release_issue () {
+  local version=$1
+  local issue
+
+  issue=$($GHCURL $K8S_GITHUB_SEARCHAPI_ROOT&q=Release+$version+Tracking+in:title+type:issue+state:open+repo:$RELEASE_TRACKING_REPO | jq -r '.items[] | (.number | tostring)' |sort -n |tail -1)
+
+  [[ -z $issue ]] && return 1
+
+  echo $issue
+}
+
+###############################################################################
+# Create a release tracking issue for posting notifications.
+# Mostly for use by --gcb since there's no other email mechanism from which
+# to send detailed html notifictions.
+# @param version RELEASE_VERSION_PRIME
+PROGSTEP[create_release_issue]="CREATE/UPDATE RELEASE TRACKING ISSUE"
+gitlib::create_release_issue () {
+  local version=$1
+  local assignee="david-mcmahon"
+  local milestone
+  local stage
+  local text
+  local cc
+  local repo
+  local issue_number
+
+  # Set the milestone and stage
+  if [[ $version =~ ${VER_REGEX[release]} ]]; then
+    milestone=${BASH_REMATCH[1]}.${BASH_REMATCH[2]}
+    stage=${BASH_REMATCH[4]/-/}
+  fi
+
+  if ((FLAGS_nomock)); then
+    repo="kubernetes/features"
+    # default to 1.9 release owner for now
+    # TODO: Can we make the assignee some sig-release group user?
+    assignee="ensonic"
```

Comment:
There's actually a user named "ensonic" but that's not me. :)

Reply:
Hah. Typo. Sorry. :)
Maybe best to parse out release owner from k/features?  Is that list staying there?

Reply:
What list do you mean?

Reply:
I use that term loosely referring to the markdown at https://github.com/kubernetes/features/blob/master/release-1.8/release-1.8.md and the others.

Reply:
Rather https://github.com/kubernetes/features/blob/master/release-1.8/release_team.md

Reply:
Hm the format for those tends to change since it's hand-written. Do we really need an assignee? Maybe it's enough to ping the right github team.

Reply:
That's certainly easier / less fragile, but I'm not sure you can assign to a team?  At least it doesn't come up in the drop down on issue assignments.

Reply:
maybe the cc to kubernetes/sig-release-members is enough and we can just leave the assignee null.  Probably not worth agonizing over at this point.

Reply:
Yeah I meant leave the assignee empty. I don't think we need it.